offline_optimization_script.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""
+Offline optimization of parameter combinations.
+
+Optimization on the server:
+
+login as ptxboa user:
+
+$ ssh ptxboa2

```

Comment:
@joAschauer Did you change anything with the ssh config? I get an error when trying to ssh: 

![grafik](https://github.com/agoenergy/ptx-boa/assets/26714648/090a19ed-3748-4d46-b9e6-700ef673bc77)

my config file only lists an ``ptxboa`` host, I did not know about `ptxboa2`